plugin/input/http/http.go

```diff
@@ -246,7 +245,7 @@ func (p *Plugin) serve(w http.ResponseWriter, r *http.Request) {
 
 	_, err := w.Write(result)
 	if err != nil {
-		stats.GetCounter(subsystemName, httpErrorCounter).Inc()
+		metrics.GetCounter(subsystemName, httpErrorCounter).Inc()
```

Comment:
is renaming of package related to linter?
I just try to see net effect of using linter in your configuration, and what I see is just this package renaming...